src/lib/snark_work_lib/work.ml

```diff
@@ -90,6 +84,21 @@ module Spec = struct
 
   let map ~f { instances; fee } =
     { instances = One_or_two.map ~f instances; fee }
+
+  let map_opt ~f_single { instances; fee } =
+    let open Option.Let_syntax in
```

Comment:
```suggestion
```

Reply:
I'll do what you requested here as well. However, opening the Let_syntax is used in a couple of places in our codebase.